File: Can you get the loop ?.js
function loop_size(node){
  let p1 = node, p2 = node.getNext();
  while (p1 !== p2)
  {
    p2 = p2.getNext().getNext();
    p1 = p1.getNext();
  }
  let len = 1;
  p1 = p1.getNext();
  while (p1 !== p2)
  {
    p1 = p1.getNext();
    len += 1;
  }
  return len;
}

File: Can you get the loop ?.py
def loop_size(node):
    p1 = node
    p2 = node.next
    while p1 != p2:
        p1 = p1.next
        p2 = p2.next.next
    res = 1
    p1 = p1.next
    while p1 != p2:
        p1 = p1.next
        res += 1
    return res
